doc/nrf/config_and_build/bootloaders/index.rst

```diff
@@ -51,7 +51,7 @@ You can find an overview of currently supported bootloaders in the table below:
      - Yes
      - :doc:`See imgtool <mcuboot:imgtool>`
      - No
-     - Yes
+     - No
```

Comment:
Was the intention to remove the column or not?
As is now, the column is still there, but says "no" for both, which is not true.

Reply:
Sorry, my intention was to leave it, will fix - thanks for catching